test/libsolidity/SolidityExecutionFramework.h

```diff
@@ -35,6 +38,8 @@
 namespace solidity::frontend::test
 {
 
+using namespace solidity::test;
+
 class SolidityExecutionFramework: public solidity::test::ExecutionFramework
```

Comment:
```suggestion
class SolidityExecutionFramework: public ExecutionFramework
```

here and everywhere else since now youre using that namespace